docs/src/main/tut/datatypes/NonEmptyList.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+title:  "NonEmptyList"
+section: "data"
+source: "core/src/main/scala/cats/data/NonEmptyList.scala"
+scaladoc: "#cats.data.NonEmptyList"
+---
+# NonEmptyList
+
+If you have had the opportunity of taking a look to [Validated](validated.html) or [IOR](ior.html), you'll find that a common case is to use `NonEmptyList` with one of these data structures.
+
+Why? Because it fits nicely in the error reporting cases. As stated by its name, `NonEmptyList` is a _specialized_ data type that will have, at least, one element. You'll find that its behavior is like a list, with the aforementioned constraint. Think of it as a `List` wrapper.
+
+The signature of the structure is as follows:
+
+```scala
+final case class NonEmptyList[+A](head: A, tail: List[A]) {
+	// Implementation elided
+}
+```
+
+The `head` of the `NonEmptyList` will be _non-empty_. Meanwhile, the `tail` can have zero or more elements.
+
+For example:
+
+```tut:book
+import cats.data.NonEmptyList
+
+val list = NonEmptyList.of("Hello")
```

Comment:
Have you thought about adding types to your examples? I think this would demonstrate the difference between `of` and `fromList` more clearly:

```scala
val list: NonEmptyList[String] = NonEmptyList.of("Hello")

val nel: Option[NonEmptyList[String]] = NonEmptyList.fromList(regularList)
```